client/src/components/UserManagement.jsx

```diff
@@ -0,0 +1,201 @@
+          label="Delete"
+          color="inherit"
+          className="actionButton"
+        />,
+      ];
+    },
+  },
+];
+
+const rows = [
+  { id: 1, name: "First Last", email: "email@email.com" },
+  { id: 2, name: "First Last", email: "email@email.com" },
+  { id: 3, name: "First Last", email: "email@email.com" },
+  { id: 4, name: "First Last", email: "email@email.com" },
+  { id: 5, name: "First Last", email: "email@email.com" },
+  { id: 6, name: "First Last", email: "email@email.com" },
+  { id: 7, name: "First Last", email: "email@email.com" },
+  { id: 8, name: "First Last", email: "email@email.com" },
+  { id: 9, name: "First Last", email: "email@email.com" },
+  { id: 10, name: "First Last", email: "email@email.com" },
+];
+
+export default function DataGridDemo() {
+  const [searchQuery, setSearchQuery] = React.useState("");
+  const [filteredRows, setFilteredRows] = React.useState(rows);
+
+  const handleSearch = (event) => {
+    const query = event.target.value;
+    setSearchQuery(query);
+
+    const filtered = rows.filter((row) =>
+      row.name.toLowerCase().includes(query.toLowerCase()),
+    );
+    setFilteredRows(filtered);
+  };
+
+  const handleFilterReset = () => {
+    setSearchQuery("");
+    setFilteredRows(rows);
+  };
+
+  return (
+    <div style={{ marginRight: 40 }}>
+      <div style={{ display: "flex", alignItems: "center" }}>
+        <ArrowBackIcon sx={{ color: "gray", marginRight: 2, marginLeft: 2 }} />
+        <Typography
+          style={{
+            fontFamily: "Arial",
+            fontSize: "34px",
+            fontWeight: 500,
+            lineHeight: "42px",
+            letterSpacing: "0.25px",
+            textAlign: "left",
+          }}
+        >
+          Admin Dashboard
+        </Typography>
+        <Button
+          sx={{
+            marginLeft: "auto",
+            marginBottom: "30px",
+            marginTop: "30px",
+            backgroundColor: "#3568E5",
+            color: "white",
+            "&:hover": {
+              backgroundColor: "#3568E5",
+              color: "white",
+            },
+          }}
+          startIcon={<AddIcon />}
+        >
+          New Users
+        </Button>
+      </div>
+
+      <Box sx={{ height: 400, width: "100%" }}>
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "row",
+            // display: "grid",
+            // gridTemplateColumns: "1fr auto",
+            gridColumnGap: "30px",
+            // height: "590px",
+            // width: "1400px"
+          }}
+        >
+          <Card sx={{ width: 235, height: 175, marginLeft: 2 }}>
+            <CardContent>
+              <Typography
+                sx={{ fontSize: 14 }}
+                color="text.secondary"
+                gutterBottom
+              >
+                Name
+              </Typography>
+              <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                <TextField
```

Comment:
Reduce the padding on this. Its currently making the text field taller than it should be
<img width="253" alt="Screenshot 2023-11-11 at 3 25 55 PM" src="https://github.com/uoftblueprint/tces/assets/58835213/cc944f0e-2835-467c-b5cd-998ef3dfa769">